flutter_local_notifications/lib/src/platform_flutter_local_notifications.dart

```diff
@@ -244,6 +244,16 @@ class AndroidFlutterLocalNotificationsPlugin
     });
   }
 
+  @override
+  Future<void> cancel(int id, {String tag}) async {
```

Comment:
API docs for this class haven't been updated to mention the tag (note: this class is also public). Currently it inherits the original API docs. I'd suggest for now to copy and paste the original docs and then make an addition to describe the tag. I can look at setting up macros for reusable docs to minimise copy and paste in a separate PR, as use of macros is something I've only found out about recently